underlay/src/main/resources/config/datamapping/aouRT/entity/observation/all.sql

```diff
@@ -4,9 +4,7 @@ SELECT
     vocabulary_id,
     concept_code,
     (CASE WHEN standard_concept IS NULL THEN 'Source' WHEN standard_concept = 'S' THEN 'Standard' ELSE 'Unknown' END) AS standard_concept
-
 FROM `${omopDataset}.concept`
-
 WHERE domain_id = 'Observation'
     AND standard_concept = 'S'
     AND vocabulary_id != 'PPI'
```

Comment:
If your going to exclude this data from observation_occurrence -> concept_class_id != 'Survey' 
Then you need to also exclude it here
```suggestion
    AND vocabulary_id != 'PPI'
    AND concept_class_id != 'Survey'
```